README.md

```diff
@@ -3,66 +3,79 @@ kindle-highlights
 
 A Ruby gem for collecting your Kindle highlights.
 
-### Install
+### Installation
 ```
 gem install kindle-highlights
 ```
 
-### Use
-```ruby
-require 'kindle_highlights'
+### Usage
 
-# pass in your Amazon credentials. Loads your books (not highlights)
-# on init, so might take a while
-kindle = KindleHighlights::Client.new("email.address@gmail.com", "password")
+1. In an interactive Ruby shell ([`irb`](http://ruby-doc.org/stdlib-2.0.0/libdoc/irb/rdoc/IRB.html)), run the first of several commands:
 
-# returns a hash of your books, keyed on the ASIN, with the title as value
-kindle.books #=>
-{
-  "B002JCSCO8" => "The Art of the Commonplace: The Agrarian Essays of Wendell Berry",
-  "B0049SPHC0" => "Calvinistic Concept of Culture, The",
-  "B003HNOB34" => "The Collected Works of William Butler Yeats (Unexpurgated Edition) (Halcyon Classics)",
-  "B000JMKZX6" => "The Essays of Arthur Schopenhauer; On Human Nature",
-  "B005CQ2ZE6" => "From the Garden to the City",
-  "B0082ZJFCO" => "The Golden Sayings of Epictetus",
-  "B000SEGEKI" => "The Pragmatic Programmer: From Journeyman to Master",
-  "B009D6AGOM" => "The Rare Jewel of Christian Contentment",
-  "B00E25KVLW" => "Ruby on Rails 4.0 Guide",
-  "B004X5RLBY" => "The Seven Lamps of Architecture",
-  "B0032UWX1O" => "The Westminster Confession of Faith",
-  "B0026772N8" => "Zen and the Art of Motorcycle Maintenance"
-}
+   `require 'kindle_highlights'`
 
-# get your highlights for a specific book by passing the ASIN
-kindle.highlights_for("B005CQ2ZE6") #=>
-[
-  {
-    "asin"          => "B005CQ2ZE6",
-    "customerId"    => "...",
-    "embeddedId"    => "From_the_Garden_to_the_City:420E805A",
-    "endLocation"   => 29591,
-    "highlight"     => "One of the most dangerous things you can believe in this world is that technology is neutral.",
-    "howLongAgo"    => "1 year ago",
-    "startLocation" => 29496,
-    "timestamp"     => 1320901233000
-  },
+1. Provide your Amazon credentials to access your book titles:
+
+   `kindle = KindleHighlights::Client.new("email.address@gmail.com", "password")`
+   
+   This might take a while.
+   
+1. Get each book (where the key is the [ASIN](http://www.amazon.com/gp/seller/asin-upc-isbn-info.html)), and its corresponding title:
+
+   `kindle.books`
+   
+   Example output is as follows:
+    ```
```

Comment:
Add the programming language after the three backticks, so that it gets highlighted properly:

```
    ```ruby
    {
        "B002JCSCO8" => "The Art of the Commonplace: The Agrarian Essays of Wendell Berry",
````